src/main/steps/applicant2Sequence.ts

```diff
@@ -211,7 +211,7 @@ const postSubmissionSequence: Step[] = [
 const hasApp2Confirmed = (data: Partial<CaseWithId>): boolean =>
   ![State.AwaitingApplicant1Response, State.AwaitingApplicant2Response, State.Draft].includes(data.state as State) &&
   data.applicant2IConfirmPrayer === Checkbox.Checked &&
-  data.applicant2StatementOfTruth === Checkbox.Checked;
+  (data.applicant2StatementOfTruth === Checkbox.Checked || data.aosStatementOfTruth === Checkbox.Checked);
```

Comment:
there's a separate sequence for respondents, so I don't think this change is needed